apps/manager_app/manager_app.h

```diff
@@ -46,4 +47,16 @@
  */
 void manager_app_main(usb_event_t usb_evt);
 
+/**
+ * @brief Return a filled instance of get_device_info_response_t.
+ */
+manager_get_device_info_response_t get_device_info(void);
+
+/**
+ * @brief Initiates the get device info flow.
+ *
+ * @param query Reference to the decoded query struct from the host app
+ */
+void get_device_info_flow(const manager_query_t *query);
```

Comment:
Let's move these functions to `manager_app_priv.h`
We should not expose the individual functions/flows of an app.

Reply:
refer https://github.com/Cypherock/x1_wallet_firmware/compare/refactor/core-framework...test/support-protobuf-auth-card?expand=1#diff-20226d7b084854f33b609e942be568155e853af7c07d575547acb1c125824ceb